src/libConsensus/ConsensusLeader.cpp

```diff
@@ -328,12 +326,21 @@ bool ConsensusLeader::ProcessMessageCommitCore(
       m_commitRedundantCounter++;
     }
 
-    // notify the waiting thread to start with subset creations and subset
-    // consensus.
-    if (m_commitCounter == m_committee.size()) {
-      lock_guard<mutex> g(m_mutexAnnounceSubsetConsensus);
-      m_allCommitsReceived = true;
-      cv_scheduleSubsetConsensus.notify_all();
+    if (NUM_CONSENSUS_SUBSETS > 1) {
+      // notify the waiting thread to start with subset creations and subset
+      // consensus.
+      if (m_commitCounter == m_committee.size()) {
+        lock_guard<mutex> g(m_mutexAnnounceSubsetConsensus);
+        m_allCommitsReceived = true;
+        cv_scheduleSubsetConsensus.notify_all();
+      }
+    } else {
+      if (m_commitCounter == m_numForConsensus) {
+        LOG_GENERAL(INFO, "Sufficient commits obtained. Required/Actual = "
```

Comment:
This log looks doesn't log the required and actual commits.

Reply:
As they are equal, we only need to display one value

Reply:
Ok, i see